src/libraries/System.Collections/tests/Generic/Stack/Stack.Generic.Tests.cs

```diff
@@ -80,7 +80,7 @@ public void Stack_Generic_Constructor_IEnumerable(EnumerableType enumerableType,
             _ = numberOfMatchingElements;
             IEnumerable<T> enumerable = CreateEnumerable(enumerableType, null, enumerableLength, 0, numberOfDuplicateElements);
             Stack<T> stack = new Stack<T>(enumerable);
-            Assert.Equal(enumerable.ToArray().Reverse(), stack.ToArray());
+            Assert.Equal(Enumerable.Reverse(enumerable.ToArray()), stack.ToArray());
```

Comment:
Same for many of these... I'm not clear on why changes here are necessary

Reply:
Same reason as the other ones - the System namespace problem.